src/us12.py

```diff
@@ -0,0 +1,46 @@
+'''
+Author:     Samantha Inneo
+Sprint:     Sprint 3
+Use Case:   Checks if parents are too old to have children
+'''
+
+import pandas as pd
+from datetime import date
+import copy
+import os
+from Gedcom import Gedcom, Family
+from us28 import sibList
```

Comment:
whats sibList? you should just be able to do something like `from us28 import us28`